gcs_oauth2_boto_plugin/oauth2_client.py

```diff
@@ -52,6 +52,11 @@
 import oauth2client.service_account
 from google_reauth import reauth_creds
 import retry_decorator.retry_decorator
+from google.auth import _helpers
```

Comment:
Should we not bump up the minor version of the library, considering we are changing the underlying auth method ?  
My understanding is that for the user, virtually nothing changes though ? Hence does not require a major version change ?

Reply:
We can do that, that's another process - https://g3doc.corp.google.com/cloud/storage/g3doc/team/sub_teams/serving/sub_teams/orchestration/sub_teams/clients/sub_teams/cli/gsutil/gcs-oauth2-boto-plugin-release.md?cl=head

We can do it as a separate change, our priority currently is to do the release process of gsutil, TODO bug for tracking https://b.corp.google.com/issues/336344983